docs/rfcs/rfc-012-autogenerated-properties.md

```diff
@@ -0,0 +1,107 @@
+# Autogenerated Properties
+
+## Problem
+We want to allow for node properties to be generated automatically, via code, when creating (and possibly updating) nodes. 
+This would solve common problems such as custom ids or creation of slugs.
+
+Related Issue: [627](https://github.com/neo4j/graphql/issues/627)
+
+## Proposed Solution
+The proposed solutions would add a new directive `@callback` an a new config option to `Neo4jGraphQL` (`callbacks`). This directive will mark some properties
```

Comment:
`@callback` directive name still feels a bit incomplete as it doesn't implies the data will be stored.
We may want to think a bit more in naming :confused: